### PR TITLE
Add Open Exchange Rate API

### PR DESCRIPTION
**Pull request by ericluria**
See https://docs.openexchangerates.org/docs/latest-json

### CODE REVIEW

File: open-exchange-rates/api.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "open-exchange-rates",
+  "base_url": "https://openexchangerates.org/api",
+
+  "models": {
+    "rate": {
+      "fields": [
+        { "name": "disclaimer", "type": "string", "description": "A disclaimer regarding usage of the API included in every response from the server." },
```
Comment (paololim) [inline open-exchange-rates/api.json]: What does this return?

**Comment by paololim** [comment]
lgtm

**Comment by mikeclimbrock** [comment]
cool! lgtm

**Comment by ericluria** [comment]
The only thing I dislike about this is naming it `open-exchange-rates`. Thoughts on naming it `oer` or `oxr`? The reason I dislike it is because the generated package name ends up looking like `io.flow.open.exchange.rates`, as opposed to `io.flow.oxr`

**Comment by mbryzek** [comment]
lgtm

**Pull request merged**